src/BetweenTheBubbles.tbasic

```diff
@@ -35,4 +37,4 @@ function tbMain()
   
   tbgl_destroyWindow
 
-end function
+end function
```

Comment:
Minor - but I would prefer to leave extra line at the end of file just to avoid seeing this red sign :D

Reply:
> Minor


not what you think, this is the result of removing the chars thinAir automatically puts at the end of a script. This and saving the file without BOM.